src/rmc.rs

```diff
@@ -205,7 +199,7 @@ impl<'a, H: Signable + Hash + Eq + Clone + Debug, MK: MultiKeychain> ReliableMul
             network_rx,
             network_tx,
             keychain,
-            node_count,
+            _node_count,
```

Comment:
Can we remove this unused field from the `ReliableMulticast` struct? We cn keep it in the constructor to not break the api.

Reply:
Done.

Reply:
Done